Makefile

```diff
@@ -409,19 +414,23 @@ ci-rp: fix-macos-vendor
 		-t $(LOCAL_ARO_RP_IMAGE):$(VERSION)
 
 	# Tag the portal build image if it exists
-	@PORTAL_IMAGE_ID=$(shell podman $(PODMAN_REMOTE_ARGS) image ls --filter label=stage=portal-build-cache-layer --noheading --format "{{.Id}}" | tail -n 1); \
+	@PORTAL_IMAGE_ID=$(shell docker image ls --filter label=stage=portal-build-cache-layer --format "{{.Id}}" | head -n 1); \
```

Comment:
I think removing `--noheading` and doing `head -n 1` will only get the header for the images. This will need to be modified.

Reply:
nevermind - it looks like `--format` doesn't print headings (?) https://docs.docker.com/reference/cli/docker/image/ls/#format